adapters/ddf-adapter/src/test/java/com/connexta/replication/adapters/ddf/csw/GetRecordsResponseConverterTest.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.adapters.ddf.csw;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+
+import com.connexta.replication.data.MetadataImpl;
+import com.thoughtworks.xstream.converters.Converter;
+import com.thoughtworks.xstream.converters.MarshallingContext;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+import com.thoughtworks.xstream.io.xml.XppReader;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import org.codice.ditto.replication.api.data.Metadata;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.xmlpull.v1.XmlPullParserFactory;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GetRecordsResponseConverterTest {
+
+  @Mock Converter converter;
+
+  @Mock HierarchicalStreamWriter writer;
+
+  @Mock MarshallingContext marshallingContext;
+
+  @Mock UnmarshallingContext unmarshallingContext;
+
+  String cswResponse =
+      " <csw:GetRecordsResponse xmlns:csw=\"http://www.opengis.net/cat/csw\">\n"
```

Comment:
✏️ If we move these into files we won't have to duplicate.

Reply:
Moved to a file